src/types.rs

```diff
@@ -1094,6 +1099,106 @@ impl ToLSP<Vec<FileEvent>> for notify::DebouncedEvent {
     }
 }
 
+pub trait ListItem {
+    fn quickfix_item(&self, lc: &LanguageClient) -> Result<QuickfixEntry>;
+    fn string_item(&self, lc: &LanguageClient, cwd: &str) -> Result<String>;
```

Comment:
I don't have any idea either on the trait naming.

For this particular method though, maybe `stringify` or `to_display`?

Reply:
Yeah that one is the hardest one. I went with `string_item` to have it named somewhat similarly to `quickfix_item`. To be honest, `to_display` sounds too Rusty to me, I feel like this name should be closer to Vim than to Rust.
The `to_` bit I'm ok with though, but I would probably rename both then, so they are named similarly, `to_quickfix_item` and `to_string_item`/`to_plaintext_item` ?

Reply:
What about `to_sel_ui_item`/`to_selui_item`? Or does that look too much like [this](https://cdn.shopify.com/s/files/1/0292/2889/2293/products/Wiggly-Worm.jpg)?